deployment/ansible/site.yml

```diff
@@ -14,7 +14,8 @@
     - { role: "ckanext-disqus" }
     - { role: "ckanext-issues" }
     - { role: "ckanext-spatial" }
-    - { role: "ckanext-deadoralive" }
+# deadoralive disabled until false positive issues are worked out
+#    - { role: "ckanext-deadoralive" }
     - { role: "ckanext-datajson" }
     - { role: "ckan-deadoralive" }
```

Comment:
Is this one OK, or should it go as well?

Reply:
Good catch

Reply:
Updated in https://github.com/azavea/opendataphilly-ckan/pull/70/commits/806d93816ec916acd078e9b54e01c2639b25b58c